### ISSUE TITLE
True color support for themes

### ISSUE THREAD

**Issue by akz92** [ranger/ranger#690]
Is there any plans of supporting True Color (24 bit) for themes? I've done some research but didn't find any info about it.

**Comment by amerlyq** [comment]
`ncurses`, driving all `ranger`drawing, has severe limitations and dependency, which I doubt will be resolved in the foreseeable future. Start from here https://gist.github.com/XVilka/8346728 and google for mailing lists with `ncurses` author. So either `ncurses` implements true color support some point in time, or `ranger` will support/migrate to some alternative (if there is any). Which way faster is hard to say on years timespan:).

**Comment by nfnty** [comment]
PRs are welcome.

**Comment by peey** [comment]
An update: the gist whose link @amerlyq posted now contains this

> Since ncurses-6.0-20180121 terminfo started to support 24-bit truecolor capability under the name of "RGB" - you need to use the "setaf" and "setab" commands to set foreground and background correspondingly.

I think this ncurses version and above is the default in bionic (18.04). I'd have checked but I'm stuck on an old version in 16.04. 

You might have to do some additional configuration though (the setaf and setab). Someone with the latest version should check if truecolor works in ranger (or how to set the required config so it works).

**Comment by toonn** [comment]
@peey Note the `ncurses` that is the problem is the python module in the stdlib, not the upstream C libraries those bind to.

**Comment by kevinhwang91** [comment]
I have a hack method to solve it using alacritty as terminal.
![2020-01-23_06-56](https://user-images.githubusercontent.com/17562139/72959421-38ebbd00-3de5-11ea-9872-5a0b2a5ffde4.png)

setting the "true color" for alacritty index colors.

```
  # vim one dark theme map
  indexed_colors:
    - { index: 16, color: '#282c34' }
    - { index: 17, color: '#3e4452' }
    - { index: 59, color: '#4b5263' }
    - { index: 60, color: '#636d83' }
    - { index: 69, color: '#528bff' }
    - { index: 73, color: '#57b7c2' }
    - { index: 75, color: '#61afef' }
    - { index: 81, color: '#80a0ff' }
    - { index: 102, color: '#828997' }
    - { index: 114, color: '#98c379' }
    - { index: 130, color: '#be5046' }
    - { index: 145, color: '#abb2bf' }
    - { index: 168, color: '#e06c75' }
    - { index: 173, color: '#d19a66' }
    - { index: 176, color: '#c678dd' }
    - { index: 180, color: '#e5c07b' }
    - { index: 233, color: '#181a1f' }
```

**Comment by tigerjack** [comment]
This is an old bug, but I was wondering if there is any update on it.

**Comment by toonn** [comment]
@tigerjack, I don't believe this can be fixed until the upstream `curses` module in the python stdlib gets fixed.
